sesman/chansrv/config.c

```diff
@@ -0,0 +1 @@
+#include "../config.c"
```

Comment:
I had problem referencing it from clipboard.c, Don't really get how it should have been done properly. 
My C has not been in use since high school, last century :)